superset/config.py

```diff
@@ -172,7 +172,11 @@ def _try_json_readsha(  # pylint: disable=unused-argument
 WTF_CSRF_ENABLED = True
 
 # Add endpoints that need to be exempt from CSRF protection
-WTF_CSRF_EXEMPT_LIST = ["superset.views.core.log", "superset.charts.api.data"]
+WTF_CSRF_EXEMPT_LIST = [
+    "superset.views.core.log",
+    "superset.charts.api.data",
+    "superset.views.core.stop_dashboard_queries",
```

Comment:
I know 2 APIs here that don't use CSRF token here: log and stop_dashboard_queries, both of 
 they use `sendbeacon` which **can't** send CSRF token. The advantage of sendbeacon vs regular POST is [here](https://benborgers.com/blog/navigator-sendbeacon): it doesn't need response. And it's pretty common for sendbeacon call go without CSRF token.

CSRF token is used to prevent malicious site from executing some transaction, like move money from your bank account to mine :)  I feel kill other ppl's queries when their dashboard is still loading, is not very dangerous.

But for `superset.charts.api.data` i am not sure why it is in the exempt list. But this is not related to this PR.

Reply:
The reason `superset.charts.api.data` is exempt is because it is only a POST due to having a large request payload that doesn't sit well with a GET.